crates/bevy_math/src/cubic_splines.rs

```diff
@@ -133,23 +141,43 @@ impl<P: VectorSpace> CubicHermite<P> {
             control_points: control_points.into_iter().zip(tangents).collect(),
         }
     }
-}
-impl<P: VectorSpace> CubicGenerator<P> for CubicHermite<P> {
+
+    /// The characteristic matrix for this spline construction.
```

Comment:
Link to an explanation of characteristic matrices please.

Reply:
I couldn't find a good reference, but I added some explanations of what the characteristic matrices are doing.